File: DesignPatterns.UI.Web/ClientApp/src/app/models/Adapter/PolicyFee.ts
export class PolicyFee
{
    constructor(policyHolderName: string, policyType: string, fee: number) {
        this.PolicyHolderName = policyHolderName;
        this.PolicyType = policyType;
        this.Fee = fee;
    }
    PolicyHolderName: string = '';
    PolicyType: string = '';
    Fee: number = 0;
}

File: DesignPatterns.AdapterPattern/Models/PolicyFee.cs
using DesignPatterns.AdapterPattern.Interfaces;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DesignPatterns.AdapterPattern.Models
{
    public class PolicyFee
    {
        public string PolicyHolderName { get; set; }
        public string PolicyType { get; set; }
        public decimal Fee { get; set; }
    }
}
